### DISCUSSION TITLE
jgit sshd ed25519 example

### DISCUSSION THREAD

**Discussion by appiepollo14** [eclipse-jgit/jgit#67] [Q&A] [answered]
Hi,

I'm trying to figure out how to do a jgit clone via ssh with an ed_25119 ssh keypair. I think I need jgit.ssh.apache in combination with sshd. I've found many examples of parts of it, but not a complete one in my opinion. Does anyone has a link or gist for this situation?

k.r. A

**Comment by davido** [comment, marked as answer]
Have you checked Gerrit Code Review [code](https://gerrit.googlesource.com/gerrit/) that uses JGit?

We have to include this additional dependency to support elliptic curve keys:

```
    maven_jar(
        name = "eddsa",
        artifact = "net.i2p.crypto:eddsa:0.3.0",
        sha1 = "1901c8d4d8bffb7d79027686cfb91e704217c3e1",
    )
```

See also this commit: [1].

[1] https://git.eclipse.org/c/jgit/jgit.git/commit/?id=db627c417779691d7fe5fa925ea63f53babbae01
